TrainingExtensions/onnx/test/python/test_seq_mse.py

```diff
@@ -420,8 +419,8 @@ def test_apply_seq_mse_for_residual_model(inp_symmetry, param_bw, loss_fn, enabl
     seq_mse = SequentialMse(model, sim, seq_params, dataloader)
     seq_mse.apply_seq_mse_algo()
 
-    weight_quantizer_conv_1 = seq_mse.sim.qc_quantize_op_dict['onnx::Conv_41']
-    weight_quantizer_conv_2 = seq_mse.sim.qc_quantize_op_dict['onnx::Conv_44']
+    weight_quantizer_conv_1 = seq_mse.sim.qc_quantize_op_dict['onnx::Conv_45']
```

Comment:
In general these references will make this unit test very brittle and likely to fail when we upgrade ONNX etc. @quic-rgite

Reply:
In a separate PR we can find a better way

Reply:
There is already a plan to re-write all the onnx test models using ONNX python API rather than export torch model to onnx. This way upgrading torch and onnx won't impact our unit-tests. We have also started using the ONNX python API approach since few months.